doc/source/conf.py

```diff
@@ -245,7 +245,6 @@
     "navbar_end": ["version-switcher", "navbar-icon-links"],
     "switcher": {
         "json_url": "https://pandas.pydata.org/versions.json",
```

Comment:
```suggestion
        "json_url": "/versions.json",
```
If you want to make this relative too, otherwise when testing the website locally, the version switcher fails due to a CORS error.

Reply:
Would you mind adding the dark theme logo? Not really sure where to change the logo based on the theme